resources/aries-test-a/droplets.tf

```diff
@@ -9,3 +9,15 @@ resource "digitalocean_droplet" "aries-test-a-nodes-farmer-relayer" {
     data.digitalocean_ssh_key.leo-key.id,
   ]
 }
+
+resource "digitalocean_droplet" "aries-farmers-network-a-nodes" {
+  image  = "ubuntu-20-04-x64"
+  name   = "aries-farmers-network-a-nodes"
+  region = "nyc3"
+  size   = "c-8"
```

Comment:
Let's use shared quad-core/8G with regular SSD instead, it really doesn't need to be very powerful for this purpose. And let's run it with stock SSD (no volume) for now, should be enough if we don't have relayer running, we'll add it later if necessary.